moulinette/actionsmap.py

```diff
@@ -368,17 +368,23 @@ def __init__(self, parser_class, namespaces=[], use_cache=True,
         for n in namespaces:
             logger.debug("loading actions map namespace '%s'", n)
 
-            if use_cache:
+            actionsmap_yml = '%s/actionsmap/%s.yml' % (pkg.datadir, n)
+            actionsmap_pkl = '%s/actionsmap/%s-%s.pkl' % (pkg.cachedir, n, os.stat(actionsmap_yml).st_size)
```

Comment:
Using mtime in addition seems reasonable at first sight : it makes the cache handling become a lot safer, so we don't need to think about it anymore, or almost.

If the added processing time is not visible, I support using it.